src/redux/reducers/index.js

```diff
@@ -24,4 +25,5 @@ export default combineReducers({
   accommodation: accommodationReducer,
   multicityTrip: multicityReducer,
   admin: adminReducer,
+  allAccomodation: viewAccomodationReducer,
```

Comment:
Would you mind placing all accommodation reducer under accommodation instead of having both `accommodation` and `allAccomodation` separately?

Reply:
If we want to use the same redux `accommodation` object to hold all the work-related, we definitely need to use the same reducer.  we 'll be having one reducer to handle all the work. Imagine if actions like create, edit, update and delete in one file. Furthermore, combine reducers does not allow duplicate key.
This is why I created separate reducer to view all accommodations, likes and ratings.
@victorkarangwa4 do you have an idea to implement it?